tests/model.test.js

```diff
@@ -0,0 +1,98 @@
+import { expect } from "chai";
+import t from "t";
+
+import { filterTree, getModelTreeByApplication } from "../src/model_features";
+
+const MODEL_TREE = {
```

Comment:
Should be imported from mode.js, probably, no?

Reply:
In principle yes, but I thought we wanted to avoid dependencies between `ade.js` and `mode.js`. I could include it in `devDependencies` though